cmd/skiff/main.go

```diff
@@ -17,8 +18,28 @@ func main() {
 		Name:  "skiff",
 		Usage: "Analyze the disk usage and directory structure of OCI images and its layers",
 		Before: func(ctx context.Context, c *cli.Command) (context.Context, error) {
+			var neededCapabilities = []capability.Cap{
+				capability.CAP_CHOWN,
+				capability.CAP_DAC_OVERRIDE,
+				capability.CAP_FOWNER,
+				capability.CAP_FSETID,
+				capability.CAP_MKNOD,
+				capability.CAP_SETFCAP,
+			}
+
 			reexec.Init()
-			unshare.MaybeReexecUsingUserNamespace(false)
+
+			capabilities, err := capability.NewPid(0)
+			if err != nil {
+				return ctx, err
+			}
```

Comment:
`NewPid` is deprecated
```suggestion
			capabilities, err := capability.NewPid2(0)
			if err != nil {
				return ctx, err
			}
			if err = capabilities.Load(); err != nil {
				return ctx, err
			}
```